pkg/synopsysctl/cmd_get.go

```diff
@@ -177,7 +177,7 @@ var getBlackDuckRootKeyCmd = &cobra.Command{
 		if err != nil {
 			return fmt.Errorf("error writing to file '%s' due to %+v", fileName, err)
 		}
-		log.Infof("successfully created the master key in file '%s' for Black Duck '%s' in namespace '%s'", fileName, blackDuckName, blackDuckNamespace)
+		log.Infof("successfully received the master key and generate the file '%s' for Black Duck '%s' in namespace '%s'", fileName, blackDuckName, blackDuckNamespace)
```

Comment:
successfully retrieved the master key and wrote it to file '%s' for ....

Reply:
done